tests/softwareupdate/test_cloud_config.py

```diff
@@ -0,0 +1,668 @@
+# coding=utf-8
```

Comment:
Great job on the testing!! 👏 🎉 

could we include maybe one more test for the user_notification_system to verify that is triggered when we wanted to be triggered?